src/EditorFeatures/Core/EditAndContinue/EditAndContinueLanguageService.cs

```diff
@@ -353,5 +359,8 @@ public async ValueTask<ManagedHotReloadUpdates> GetUpdatesAsync(CancellationToke
                 return null;
             }
         }
+
+        internal void LogPotentiallyMissingChecksumAlgorithm(string solutionPath)
+            => ImmutableInterlocked.Update(ref _solutionsWithMissingChecksumAlgorithm, static (set, solutionPath) => set.Add(solutionPath), solutionPath);
```

Comment:
Did you want to do individual projects if only certain projects have it? Or to avoid having to guess which solutions contain which projects?

Reply:
I don't think it matters. We just need an indication from a dump that this happened so that I don't need to guess why EnC does not work when looking into VS Feedback.

Reply:
Pinning SDK in global.json is per solution, not per project. If someone is hacking individual projects then it doesn't matter which one was it.